Sources/FigmaExport/Input/Params.swift

```diff
@@ -92,11 +93,13 @@ struct Params: Decodable {
         }
 
         struct Typography: Decodable {
+			let fontSystem: FontSystem?
             let fontSwift: URL?
-            let labelStyleSwift: URL?
+            let textStyleSwift: URL?
+			let labelStyleSwift: URL?
             let swiftUIFontSwift: URL?
-            let generateLabels: Bool
-            let labelsDirectory: URL?
+            let generateStyles: Bool
+            let stylesDirectory: URL?
             let nameStyle: NameStyle
         }
```

Comment:
indentation is not consistent here

Reply:
ahh good one, its difficult on the project since the default is spaces, so I need to change this for every file I'm working on.